dev-packages/node-integration-tests/suites/tracing/tedious/docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: '3.9'
+
+services:
+  db:
+    image: mcr.microsoft.com/mssql/server:2022-latest
+    restart: always
+    container_name: integration-tests-tedious
+    ports:
+      - '1433:1433'
+    environment:
+      ACCEPT_EULA: 'Y'
+      MSSQL_SA_PASSWORD: 'TESTing123'
```

Comment:
Password must pass this [criteria](https://hub.docker.com/r/microsoft/mssql-server).